Formula/folly.rb

```diff
@@ -55,6 +55,8 @@ def install
         -DFOLLY_USE_JEMALLOC=OFF
       ]
 
+      args += %w[-DFOLLY_HAVE_INT128_T=ON] if OS.mac?
```

Comment:
Could you add a comment documenting what this is needed for?

Also, is this flag portable to ARM macOS?

Reply:
As not all targets support it, this is not default. As I searched, not all compiler versions have the way to automatically detect it. I think it is why Folly doesn't do it.

Reply:
Sorry, I should've been clearer: it is clear that this flag enables 128-bit integer features. What isn't clear at all is why we would want to do this, and that's what the comment needs to address.

Reply:
On the targets where 128-bit integer are available, the caller of Folly library can call its methods with 128-bit integers. In the case, if we don't enable this option with Folly, there will be compilation error like "error: no matching function for call to ...". For example the [issue](https://github.com/facebookincubator/velox/issues/360) we encountered when building Velox.

Reply:
Does it make sense for you?

Reply:
So why isn't this enabled by default? Why do we want to deviate from the settings the Folly devs chose here?